github/util.go

```diff
@@ -91,6 +91,17 @@ func buildChecksumID(v []string) string {
 	return fmt.Sprintf("%x", bs)
 }
 
+func expandNestedSet(m map[string]interface{}, target string) []string {
```

Comment:
🧹  Moved from `github/util_v4.go` because this isn't v4 api specific (as far as I can tell) and looked similar to the `expandStringList` func that was here, too.